.github/workflows/test-testworkflows.yml

```diff
@@ -0,0 +1,35 @@
+name: Test Testworkflows
```

Comment:
Schau auch mal in unsere anderen Repo, wie wir dort die Workflows aufsetzen.

z.B.: 
- https://github.com/agentic-layer/showcase-cross-selling/blob/main/.github/workflows/check.yml
- https://github.com/agentic-layer/agent-samples/blob/main/.github/workflows/publish.yml

Außerdem in die workflows dieses Repos. Schau mal, dass sie konsistent zueinander sind.

Hinweis: `poe` oder multi project builds brauchen wir hier nicht.